pkg/controllers/jobflow/jobflow_controller_action.go

```diff
@@ -260,10 +260,16 @@ func (jf *jobflowcontroller) loadJobTemplateAndSetJob(jobFlow *v1alpha1flow.JobF
 
 	*job = v1alpha1.Job{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        jobName,
-			Namespace:   jobFlow.Namespace,
-			Labels:      map[string]string{CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName)},
-			Annotations: map[string]string{CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName)},
+			Name:      jobName,
+			Namespace: jobFlow.Namespace,
+			Labels: map[string]string{
+				CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName),
+				CreatedByJobFlow:     jobFlow.Name,
+			},
+			Annotations: map[string]string{
+				CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName),
+				CreatedByJobFlow:     jobFlow.Name,
```

Comment:
Thanks for your contribution! Is it neened to add both label and annotation?

Reply:
I have considered this issue. I refer to the fact that when creating vcjob, both annotation and label are added at the same time, so I think jobflow also needs to be added at the same time.
```bash
root@VM-0-17-ubuntu:/home/ubuntu# kubectl get pods test-a-default-nginx-0 -oyaml
apiVersion: v1
kind: Pod
metadata:
  annotations:
    scheduling.k8s.io/group-name: test-a-4e75ec64-feca-4489-aaef-f8a5bc1a2af9
    volcano.sh/job-name: test-a
    volcano.sh/job-version: "0"
    volcano.sh/queue-name: default
    volcano.sh/task-index: "0"
    volcano.sh/task-spec: default-nginx
    volcano.sh/template-uid: test-a-default-nginx
  creationTimestamp: "2024-05-27T03:07:34Z"
  labels:
    volcano.sh/job-name: test-a
    volcano.sh/job-namespace: default
    volcano.sh/queue-name: default
    volcano.sh/task-index: "0"
    volcano.sh/task-spec: default-nginx
  name: test-a-default-nginx-0
  namespace: default
  ownerReferences:
  - apiVersion: batch.volcano.sh/v1alpha1
    blockOwnerDeletion: true
    controller: true
    kind: Job
    name: test-a
    uid: 4e75ec64-feca-4489-aaef-f8a5bc1a2af9
  resourceVersion: "350123"
  uid: 9480b917-6c98-4232-a763-990a15f8e83a

```

Reply:
I think adding the job name to the label or annotation has no impact. We can be consistent with the behavior of the job, just like the Pod's label and annotation also have the name of the job.

Reply:
I have the same idea with @hwdef  cc @Monokaix